src/adagrams.js

```diff
@@ -1,7 +1,86 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    // Amount of letters available for the game
+    const allLetters = {
+      A: 9, B: 2, C: 2, D: 4, E: 12,
+      F: 2, G: 3, H: 2, I: 9, J: 1,
+      K: 1, L: 4, M: 2, N: 6, O: 8,
+      P: 2, Q: 1, R: 6, S: 4, T: 6,
+      U: 4, V: 2, W: 2, X: 1, Y: 2, Z: 1
+    };
+
+    // Array to store the letters from the hash table
+    let letterArray = []
+    for (let letter in allLetters) {
+      for (let char = 0; char < allLetters[letter]; char++) {
+        letterArray.push(letter);
+      };
+    };
+    
+    // Get 10 random single-string letters
+    letterArray.sort(() => Math.random() - Math.random());
```

Comment:
Woah. Nice, concise shuffle.

Reply:
You might also consider choosing 10 random letters rather than shuffling the entire pool.